system/lang/en.json

```diff
@@ -567,7 +567,7 @@
       "TypeCharacter": "Characters",
       "TypeFoe": "NPCs",
       "TypeLocation": "Locations",
-      "TypeSite": "Delve Sites",
+      "TypeDelveSite": "Delve Sites",
```

Comment:
Do these need to change back because we're rolling back the `delve-site` change? Not sure what Foundry expects here. I don't have a preference, since it seems like we're linking them manually elsewhere.